Library/Formula/unrar.rb

```diff
@@ -15,6 +15,11 @@ class Unrar < Formula
   def install
     system "make"
     bin.install 'unrar'
+    system "make", "clean"
+
+    system "make", "lib"
+    mv "./libunrar.so", "./libunrar.dylib"
```

Comment:
Ah, you already have. Great! If possible, move the notation back inside ` def install ` and just above the lib rename element. Helps highlight the context a little better.